cmd/skopeo/copy.go

```diff
@@ -78,6 +79,7 @@ See skopeo(1) section "IMAGE NAMES" for the expected format
 	flags.BoolVar(&opts.preserveDigests, "preserve-digests", false, "Preserve digests of images and lists")
 	flags.BoolVar(&opts.removeSignatures, "remove-signatures", false, "Do not copy signatures from SOURCE-IMAGE")
 	flags.StringVar(&opts.signByFingerprint, "sign-by", "", "Sign the image using a GPG key with the specified `FINGERPRINT`")
+	flags.StringVar(&opts.signPassphraseFile, "sign-passphrase-file", "", "Path pointing to a passphrase file when signing")
```

Comment:
Non-blocking? “Path of a file” might be sufficient, `--help` benefits from shorter text. “When signing” doesn’t say what it _is_ … “for the --sign-by key”? That’s fairly long.

In all the copies.

Reply:
Well, or just something like “file that contains a passphrase for the --sign-by key”. The concept of paths can probably be safely assumed to be understood :)